### DISCUSSION TITLE
Add native Apple Silicon binaries to release (Mac M1, M2, etc.)

### DISCUSSION THREAD

**Discussion by iwclibro** [pawelsalawa/sqlitestudio#4900] [Q&A] [answered]
Are there any plans to offer OSX downloads using native Apple Silicon binaries, so Rosetta is no longer necessary?

**Comment by tuffnatty** [comment, marked as answer]
You can get and test nightly Universal and Arm64 builds: https://github.com/pawelsalawa/sqlitestudio/actions/runs/6845148624

**Reply by irmo-de** [reply]
Great, thanks for your dedication, tuffnatty!

**Reply by GwynethLlewelyn** [reply]
Sadly, that link won't work any longer...
